en/concept/geometry_files.md

```diff
@@ -0,0 +1,80 @@
+# Geometry Files
+
+- Associated dev guide issue: https://github.com/PX4/PX4-Devguide/issues/349
+- Geometry files are found here: https://github.com/PX4/PX4-Autopilot/tree/master/src/lib/mixer/MultirotorMixer/geometries
+- They are TOML files
+
+@[youtube](https://youtu.be/orvng_11ngQ?t=1080) - Control Allocation - 18 minute in.
+
+[Slide 14 has info](https://static.sched.com/hosted_files/px4developersummitvirtual2020/1b/PX4%20Developer%20Summit%202020%20-%20Overview%20of%20multicopter%20control%20from%20sensors%20to%20motors.pdf)
+
+<!-- stuff just copied from the slide
+In a nutshell
+Desired forces
+and torques (m)
+?
+Actuator
+outputs (u)
+Control allocation
+matrix
+Actuator
+effectiveness
+matrix (B)
+Inverse
+Control
+allocation
+Matrix (P=B
++
+Moore-Penrose )
+-->
+
+
+## How to add a new geometry
+
+1. Create new TOML geometry file in [/src/lib/mixer/MultirotorMixer/geometries](https://github.com/PX4/PX4-Autopilot/tree/master/src/lib/mixer/MultirotorMixer/geometries).
+   For example "foo.toml".
+   with a new key (e.g.: key = "4fo") and add to **CMakeLists.txt** <!-- which one? -->
+
+2. In ROMFS/px4fmu_common/Create new mixer file **mixers/foo.main.mix** with a line containing the new key:
+   ```
+   R 4fo 10000 10000 10000 0  
```

Comment:
Presentation indicates that the numbers are just legacy and that everything needed is compiled from TOML . Is that correct?
So what if they are omitted? Or are all values ignored?

Reply:
Yes, it was for legacy compatibility and we now removed those, so you should just write `R: 4fo` now. The rest is ignored